mlflow/pytorch/__init__.py

```diff
@@ -876,6 +876,7 @@ def load_state_dict(state_dict_uri, **kwargs):
 @autologging_integration(FLAVOR_NAME)
 def autolog(
     log_every_n_epoch=1,
+    log_every_n_step=False,
```

Comment:
@adamreeve Thanks for the PR! Curious why the default value is False. Can we set it to `None`?

Reply:
Not sure why I went with `False` to be honest, `None` is probably more conventional, I'll change it.